zerver/lib/actions.py

```diff
@@ -5700,10 +5700,11 @@ def do_update_message(
 
         if new_stream is not None:
             assert stream_being_edited is not None
+            changed_msg_ids = [msg.id for msg in changed_messages]
```

Comment:
Changed this to `changed_message_ids` as part of our general no abbreviations policy.